mods/ra/rules/infantry.yaml

```diff
@@ -511,6 +511,7 @@ HIJACKER:
 	Captures:
 		CaptureTypes: vehicle
 		PlayerExperience: 50
+		CaptureRange: 1536
```

Comment:
This is a bit big, IMO.  0c512 would go to the edge of the cell and roughly match the visual and hitshape size of the actor.

Reply:
I thought so too but that didn't work for moving actors in practice. I kept doubling the distance until things worked and realized that, as the distance is computed from center-to-center between the actors, we need at least 1024 to cover NWSE cells and x1.414 to cover NE SE SW NW cells.

Reply:
The concern is that will this large number look bad when the hijacker captures a stopped vehicle? The answer was no, as MoveNextTo activity forces the hijacker to move right next to the actor as closest as it can.

Reply:
My concern here is that you are effectively disabling the range check completely, making it teleport immediately inside the actor as soon as MoveAdjacentTo completes.

Remember that ranges are measured from the middle, so the maximum center-edge distance is actually sqrt(2) * 512.

Reply:
Actually, I didn't see any teleporting into the actor when I tested this a minute ago, at least visually it still moved to the target center, even with a value of 5120.

Reply:
This raises a rather important flaw / confusion here: the range check only applies *after* `MoveIntoTarget` completes, and is effectively saying "if the target has moved less than X world units since we started enter it, then complete the capture anyway".  This is not a capture range as we would normally consider it.

This is really covering over a bug in `MoveIntoTarget`, where it doesn't *actually* move into the target, but into the location that was occupied by the target when the activity was first queued.  A better fix here would be to change that to lerp/drag towards the `target.CenterLocation` so that the captor is always at the right position by the end of the move.  This then removes the need to define a potentially misleading range.